.github/scripts/download-toolchain.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/config.sh
+
+echo "::group::Download toolchain"
+    RELEASE_NAME=`gh release view --json name --jq .name`
+    rm -f $ARTIFACT_PATH/$TOOLCHAIN_PACKAGE_NAME
```

Comment:
what if $RELEASE_NAME is empty?

Reply:
I am not sure I understand the merrit of the question. The relase name will be empty only when there is no release. In that case the download fails witch is IMO the expected behavior.

Reply:
maybe it is better to check if some release has been found